src/bootstrap/compile.rs

```diff
@@ -1102,6 +1102,10 @@ impl Step for Assemble {
             let src_exe = exe("lld", target_compiler.host);
             let dst_exe = exe("rust-lld", target_compiler.host);
             builder.copy(&lld_install.join("bin").join(&src_exe), &libdir_bin.join(&dst_exe));
+            // we only support gcc-lld for x86_64-linux right now
+            if target_compiler.host.triple == "x86_64-unknown-linux-gnu" {
+                builder.copy(&libdir_bin.join(&dst_exe), &libdir_bin.join("ld"));
+            }
```

Comment:
I think it should be fine to just copy it for all host targets.

Reply:
does gcc run on windows? if so, does it look for ld.exe instead of ld?

Reply:
Yeah, it can run on windows. It probably indeed looks for ld.exe then.